README.md

```diff
@@ -5,17 +5,44 @@ Bogus documentation page.
 
 ## Requirements (per assignment)
 To make a **documentation page** featuring:
-- 
+- A < main> element with id="main-doc" holding the main content
```

Comment:
To add code inline into .md files you just need to surround it by bacticks (`)
```suggestion
- A `<main>` element with id="main-doc" holding the main content
```
The other way is with triple backtics. Check the documentation for [code and syntax highlighting](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) to see examples of it. 

Please, use this throughout the document.